pkgs/applications/misc/gxmessage/default.nix

```diff
@@ -0,0 +1,20 @@
+{stdenv, fetchurl, gnome3, intltool, pkgconfig, texinfo}:
+
+stdenv.mkDerivation rec {
+  name = "gxmessage";
+  version = "3.4.3";
+
```

Comment:
I'll modify it, but is there any reason to push until the travis-ci problem is addressed?